src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsAdditionalInfo.tsx

```diff
@@ -24,17 +26,28 @@ export class ArtworkDetailsAdditionalInfo extends React.Component<
       publisher,
       manufacturer,
       image_rights,
+      isBiddable,
       framed,
       signatureInfo,
       conditionDescription,
       certificateOfAuthenticity,
+      saleArtwork,
     } = this.props.artwork
 
     const listItems = [
       {
         title: "Condition",
-        value: conditionDescription ? conditionDescription.details : null,
+        value:
+          isBiddable && sd.ENABLE_REQUEST_CONDITION_REPORT ? (
+            <RequestConditionReport
+              saleArtworkID={saleArtwork.internalID}
+              relay={this.props.relay}
```

Comment:
Rather than passing the `relay` prop around, you can get `relayEnvironment` directly from the `useSystemContext` call.